steps/twitter-message/0.0.3/step.yml

```diff
@@ -0,0 +1,78 @@
+title: twitter-message
```

Comment:
Please avoid using hyphen (`-`) separated words as step title, this will be visible in the graphical workflow editor.
Based on some already existing notification step titles (`Send a Slack message`, `Send HipChat message`, `Send Email with Mailgun`) i would recommend something like: `Send message on twitter/Send twitter message/...`

Reply:
Ok but i prefer using the same terms as twitter... Update twitter status... That is the right term used by twitter.